api.js

```diff
@@ -0,0 +1,24 @@
+var url = 'https://content.guardianapis.com/search?api-key=fed10329-e5b5-4ed3-afda-6ac7a6d00245'
```

Comment:
Under the "Guardian API example" section of the challenge README, there is another API that you should use for this challenge. 

It's considered bad practice to store API keys in the frontend - can you think of a reason why this might be?